config/config-feature-flags.yaml

```diff
@@ -84,4 +84,4 @@ data:
   enable-api-fields: "stable"
   # Setting this flag to "true" scopes when expressions to guard a Task only
   # instead of a Task and its dependent Tasks.
```

Comment:
NIT: we could update this comment as well

Reply:
good catch, fixed in https://github.com/tektoncd/pipeline/pull/4584, thanks 😀